tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2019 The xridge kubestone contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"os/exec"
+
+	"github.com/firepear/qsplit"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/xridge/kubestone/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	k8sscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	perfv1alpha1 "github.com/xridge/kubestone/api/v1alpha1"
+)
+
+const (
+	iperf3SampleCR = "../../config/samples/perf_v1alpha1_iperf3.yaml"
+	e2eNamespace   = "kubestone-e2e"
+)
+
+var restClientConfig = ctrl.GetConfigOrDie()
+var client ctrlclient.Client
+var ctx = context.Background()
+var scheme = runtime.NewScheme()
+
+func init() {
+	_ = k8sscheme.AddToScheme(scheme)
+	err := perfv1alpha1.AddToScheme(scheme)
+	if err != nil {
+		panic(err)
+	}
+
+	client, err = ctrlclient.New(restClientConfig, ctrlclient.Options{Scheme: scheme})
+	if err != nil {
+		panic(err)
+	}
+}
+
+func run(command string) (stdout, stderr string, err error) {
+	commandArray := qsplit.ToStrings([]byte(command))
+	cmd := exec.Command(commandArray[0], commandArray[1:]...)
+	var stdOut bytes.Buffer
```

Comment:
[minor] In one line: `var stdOut, stdErr bytes.Buffer`

Reply:
done